actions/helm-gke-setup/action.yaml

```diff
@@ -53,6 +53,12 @@ runs:
       with:
         version: ${{ inputs.kubectl-version }}
 
+    - name: Install kfilt
+      shell: bash
+      run: |
+        curl -L "https://github.com/ryane/kfilt/releases/download/v0.0.7/kfilt_0.0.7_linux_amd64" -o /usr/local/bin/kfilt
```

Comment:
Why not use a docker image that has kfilt available?

Reply:
Yes, building a docker image would be the next step. I would prefer to move this into this issue: https://github.com/bakdata/ci-templates/issues/16 
Is that fine?

By moving it, I can first finish building the docker release ci template (see this branch: https://github.com/bakdata/ci-templates/blob/feature/docker-release-workflow/.github/workflows/docker-release.yaml) and build out proper ci-images here: https://github.com/bakdata/ci-images/tree/main

Reply:
Please update the issue and leave a not on what to install. Also leave a TODO here to remove the installation

Reply:
Done